app/assets/javascripts/components/articles/add_available_articles.jsx

```diff
@@ -64,6 +65,7 @@ const AddAvailableArticles = createReactClass({
   },
 
   render() {
+    const articlesOrItems = ArticleUtils.articlesOrItems(this.props.course.home_wiki.project);
```

Comment:
This should use the project from `props` rather than the home_wiki, so that it automatically changes terminology to match the selected wiki.